### PR TITLE
Add custom pluralisation for the new Product Safety Alerts, Reports and Recalls

### PR DESCRIPTION
**Pull request by callumknights**
Add custom pluralisation for the new Product Safety Alerts, Reports and Recalls
specialist finder.

Trello: https://trello.com/c/ojsFViiX/2746-create-new-specialist-document-and-finder-product-safety-alerts-unsafe-products-and-recalls-3-5

⚠️ This repo is Continuously Deployed: make sure you [follow the guidance](https://docs.publishing.service.gov.uk/manual/development-pipeline.html#merge-your-own-pull-request) ⚠️

### CODE REVIEW

File: config/locales/en.yml [hunk 1]
```diff
@@ -224,6 +224,9 @@ en:
       publication:
         one: Publication
         other: Publications
+      product-safety-alert-report-recalls:
```
[the remark this answers is not in the document]
Comment (callumknights) [reply config/locales/en.yml:227]: Thanks for pointing that out Ben, getting a bit lost in singular vs plural at the moment

[the remark this answers is not in the document]
Comment (callumknights) [reply config/locales/en.yml]: Sorted 👍

**Review by benjamineskola — APPROVED** [review]

**Comment by callumknights** [comment]
There is actually some prior art for this with the `medical-safety-alert` key, so I'm wondering if a comment in the code is necessary. Should anyone think that's a mistake in future it's easy enough to find this PR in Github. I'll say for anyone else reading this in future that the reason for this pluralisation change is that there should be no reason in the frontend so display the name of this finder in the singular, as it doesn't read correctly. There are a few instances where it's named in the singular in specialist-publisher in the code, or even half plurals where just "recalls" is plural, but that's to work around instances where model's are called using things like `document_type.pluralise.new` which without custom inflections in the locales (which specialist-publisher doesn't have) just adds a trailing "s".

**Comment by benjamineskola** [comment]
I'm inclined to think it's safe, yeah, @callumknights — looking at it, I can see it would be awkward if used singularly ("product safety alert, report, _or_ recall"?) and also that the automatic pluralisation would get it wrong (because it wouldn't pluralise "alert" or "report", only "recall").

**Comment by nicholsj** [comment]
Ok, thanks both. Happy to go with your recommendations - in any case, this conversation adds some detail!

**Pull request merged**